xjc-maven-plugin/src/main/java/com/github/davidmoten/xjc/XjcMojo.java

```diff
@@ -145,6 +163,37 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         return command;
     }
 
+    private Stream<String> getPluginRuntimeDependencyEntries()
```

Comment:
This is great, just what I was after, thanks!

Reply:
Thanks!